service/src/main/java/tds/session/web/advice/ExceptionAdvice.java

```diff
@@ -0,0 +1,46 @@
+package tds.session.web.advice;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.method.annotation.MethodArgumentTypeMismatchException;
+
+import tds.common.web.exceptions.NotFoundException;
+import tds.common.web.resources.ExceptionMessageResource;
+
+@ControllerAdvice
+class ExceptionAdvice {
```

Comment:
Moving this to common will take a little more investigation/work to get Spring to recognize that the advice is in another shared dependency.  I'll come back to this later in the sprint.